ApplicationCode/GeoMech/GeoMechDataModel/RigFemPartResultsCollection.cpp

```diff
@@ -3060,3 +3297,31 @@ void findReferenceElementForNode( const RigFemPart& part, size_t nodeIdx, size_t
         }
     }
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+std::vector<std::string> RigFemPartResultsCollection::getStressVectors()
```

Comment:
getStressComponentNames() ? We're not getting the vectors but the component names.

Reply:
Fixed.